### DISCUSSION TITLE
Is there any reason why "reconnectable" is present on type "Edge" but not on type "BaseEdge".

### DISCUSSION THREAD

**Discussion by gousali522** [xyflow/xyflow#4689] [Q&A]
I was having custom Edge in which I need to use "reconnectable" value but couldn't get it from props as  "reconnectable" is not present in "BaseEdge" or "EdgeProps".

**Comment by bcakmakoglu** [comment]
Good question, not sure if there's a particular reason this prop was omitted (at least for EdgeProps).

You could always just fetch the necessary edge object from the internal state using `useReactFlow`.

```ts
const { getEdge } = useReactFlow()

const edge = getEdge(edgeId)

const isReconnectable = !!edge.reconnectable
```

**Reply by gousali522** [reply]
I guess fetching 'reconnectable' data of the edge from `getEdge` should serve the purpose. 

Thanks for the response @bcakmakoglu.
